db/migrate/20131231134936_create_schools.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.integer :score
 
-      t.timestamps
+      t.timestamps null: true
```

Comment:
I went with the "stupid default" which allows SQL timestamp fields to be null.  This was the existing behavior, but probably not what you expect.